Library/Formula/fish.rb

```diff
@@ -36,9 +36,16 @@ def caveats; <<-EOS.undent
       chsh -s #{HOMEBREW_PREFIX}/bin/fish
     to make fish your default shell.
 
-    If you are upgrading from an older version of fish, you should now run:
+    And optionally:
+      fish_config
```

Comment:
Is this specific to Homebrew? We try to avoid including general advice since it can become outdated and the more information in `caveats`, the less users actually read.